src/llmq/utils.h

```diff
@@ -85,6 +85,8 @@ Consensus::LLMQType GetInstantSendLLMQType(bool deterministic);
 bool IsDIP0024Active(const CBlockIndex* pindex);
 bool IsV19Active(const CBlockIndex* pindex);
 const CBlockIndex* V19ActivationIndex(const CBlockIndex* pindex);
+bool IsV20Active(const CBlockIndex* pindex);
+static bool IsInstantSendLLMQTypeShared();
```

Comment:
Don't understand why after the rebase this is marked as changed :thinking_face:

Reply:
because this line doesn't exist in `develop`, it was removed in https://github.com/dashpay/dash/commit/2b61dd8673b318bf75bce47947f4fbe30a965195#diff-2510710f1043f1c1fc63219328230e84ac5269f910fef46ad3e7f60c57d45ee2L98